src/PersistStore.ts

```diff
@@ -153,7 +153,11 @@ export class PersistStore<T, P extends keyof T> {
             let propertyData = target[propertyName];
             
             if (propertyData instanceof ObservableMap) {
-              propertyData = Array.from(propertyData.entries());
+              const mapArray: any = [];
+              propertyData.forEach((v, k) => {
+                mapArray.push([k, toJS(v)])
+              })
+              propertyData = mapArray;
```

Comment:
This would be cleaner ;)

```ts
propertyData = propertyData.map((v, k) => [k, toJS(v)]);
```

Reply:
propertyData is MAP and doesn't have a `map` function